lib/maven.js

```diff
@@ -7,12 +7,14 @@ async function updateVersionInPomXml (logger, versionStr) {
   logger.log(`Updating pom.xml to version ${versionStr}`)
   await exec(
     'mvn',
-    ['versions:set', 'versions:commit', `-DnewVersion=${versionStr}`]
+    ['versions:set', '-DgenerateBackupPoms=false', `-DnewVersion=${versionStr}`]
```

Comment:
Sorry I didn't catch this earlier, but I'm not sure I would be able to interpret why this flag `DgenerateBackupPoms` is being set without a lot of google searches. It might be worth adding a comment here explaining.

Reply:
I added a comment about this command.